models/account_abstraction/erc4337/base/account_abstraction_erc4337_base_v0_7_account_deployed.sql

```diff
@@ -0,0 +1,21 @@
+{{ config
+(
+    alias = 'v0_7_account_deployed',
+    
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['userop_hash', 'tx_hash']
+)
+}}
+
+
+-- macros/models/sector/erc4337
+{{
+    erc4337_account_deployed(
+        blockchain = 'base',
+        version = 'v0.7',
+        account_deployed_evt_model = source('erc4337_base','EntryPoint_v0_7_evt_AccountDeployed'),
```

Comment:
Please add all new sources used in the corresponding source file:
https://github.com/duneanalytics/spellbook/blob/main/sources/account_abstraction/erc4337/base/account_abstraction_erc4337_base_sources.yml

Reply:
Have added them here:

https://github.com/duneanalytics/spellbook/pull/5560